core/jazz_acl/swagger/swagger.json

```diff
@@ -0,0 +1,1596 @@
+              }
+            },
+            ".*\"BadRequest\".*": {
+              "statusCode": "400",
+              "responseTemplates": {
+                "application/json": "$input.path('$.errorMessage')"
+              },
+              "responseParameters": {
+                "method.response.header.Access-Control-Allow-Headers": "'Content-Type,X-Amz-Date,Authorization,X-Api-Key'",
+                "method.response.header.Access-Control-Allow-Methods": "'*'",
+                "method.response.header.Access-Control-Allow-Origin": "'*'"
+              }
+            },
+            "default": {
+              "statusCode": "200",
+              "responseTemplates": {
+                "application/json": "{\"data\" : $input.json('$.data'),\"input\" : $input.json('$.input')}"
+              },
+              "responseParameters": {
+                "method.response.header.Access-Control-Allow-Headers": "'Content-Type,X-Amz-Date,Authorization,X-Api-Key'",
+                "method.response.header.Access-Control-Allow-Methods": "'*'",
+                "method.response.header.Access-Control-Allow-Origin": "'*'"
+              }
+            },
+            ".*\"Unauthorized\".*": {
+              "statusCode": "401",
+              "responseTemplates": {
+                "application/json": "$input.path('$.errorMessage')"
+              },
+              "responseParameters": {
+                "method.response.header.Access-Control-Allow-Headers": "'Content-Type,X-Amz-Date,Authorization,X-Api-Key'",
+                "method.response.header.Access-Control-Allow-Methods": "'*'",
+                "method.response.header.Access-Control-Allow-Origin": "'*'"
+              }
+            },
+            ".*\"Forbidden\".*": {
+              "statusCode": "403",
+              "responseTemplates": {
+                "application/json": "$input.path('$.errorMessage')"
+              },
+              "responseParameters": {
+                "method.response.header.Access-Control-Allow-Headers": "'Content-Type,X-Amz-Date,Authorization,X-Api-Key'",
+                "method.response.header.Access-Control-Allow-Methods": "'*'",
+                "method.response.header.Access-Control-Allow-Origin": "'*'"
+              }
+            },
+            ".*\"NotFound\".*": {
+              "statusCode": "404",
+              "responseTemplates": {
+                "application/json": "$input.path('$.errorMessage')"
+              },
+              "responseParameters": {
+                "method.response.header.Access-Control-Allow-Headers": "'Content-Type,X-Amz-Date,Authorization,X-Api-Key'",
+                "method.response.header.Access-Control-Allow-Methods": "'*'",
+                "method.response.header.Access-Control-Allow-Origin": "'*'"
+              }
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "acl"
+        ],
+        "summary": "delete all policies(permissions) for users on the service",
```

Comment:
Few paths have `description`& others have `summary`.  Can we use one of these consistently?

Reply:
will be description